test/test_grid_dofhandler_vtk.jl

```diff
@@ -94,3 +94,40 @@ end # of testset
 if OVERWRITE_CHECKSUMS
     close(csio)
 end
+
+
+# right = Vec{2, Float64}(ntuple(x->1.5, dim))
+# left = -right
+
+@testset "Grid utils" begin
```

Comment:
Added some additional tests (to increase test coverage) when I added the `compute_vertex_values`